Fix:
error: 'TARGET_OS_IPHONE' is not defined, evaluates to 0 [-Werror,-Wundef-prefix=TARGET_OS_]
Reported to the developer by email.
--- src/system/VectorOps.h.orig	2022-01-20 04:28:07.000000000 -0600
+++ src/system/VectorOps.h	2022-01-24 17:16:37.000000000 -0600
@@ -39,6 +39,7 @@
 #ifdef HAVE_VDSP
 #include <Accelerate/Accelerate.h>
 #include <alloca.h>
+#include <TargetConditionals.h>
 #endif
 
 #include <cstring>
